src/app.ts

```diff
@@ -18,11 +20,25 @@ const seasonsController = new SeasonsController(dataProvider);
 const app = express();
 const port = configuration.port;
 
+const options = {
+    apis: ["./dist/Routes/*.js"],
```

Comment:
not sure that I like having the /dist/ path here.  Is there anyway we can infer this?

Reply:
Maybe, not sure how to though without making an explicit mapping.